### ISSUE TITLE
Allow user created host file on top of bromite.org's host file

### ISSUE THREAD

**Issue by anonymous5555522222** [bromite/bromite#2153]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md)
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have searched [existing issues](https://github.com/bromite/bromite/issues) for my feature request. This is a new issue (NOT a duplicate) and is not related to another issue.

### Is your feature request related to privacy?

Yes

### Is there a patch available for this feature somewhere?

No.

### Describe the solution you would like

I would like to block certain domains that I personally do not want my phone to connect to.
I have done this on my PC running Windows by editing the "hosts" file.
Therefore I tried to use a very useful app called DNS66, which did block these domains.
Despite the domains had been blocked on Firefox successfully, sadly Bromite seems to bypass DNS66.
However I realized that I could add these domains on Bromite's "ad blocking" settings.
The problem is that firstly, the file needs to be uploaded to a public server (in case I want to use mobile data for example), which I really do not want to do because I would rather not share what domains I would like to have blocked...
Secondly, I would have to constantly update this file with domains that have ads or malware.
So please, let us users add domains on top of the owns that are present on the file being hosted on Bromite.org!

### Describe alternatives you have considered

An alternative solution would be to not allow Bromite to bypass DNS66, but I am not sure if this is possible.
Thanks to all the developers of Bromite! It is my favorite browser on Android.

**Comment by csagan5** [comment]
This feature would be equivalent of an editor for the filters embedded in browser; that is not something trivial to add.

If you disable DoH as @arm64-v9a you will probably achieve what you want.

**Comment by anonymous5555522222** [comment]
Thank you both, disabling DoH seems to fix it!
